README.rst

```diff
@@ -126,6 +125,44 @@ Comprehensive documentation is available here_.
 
 .. _here: https://pi-top-pi-top-python-sdk.readthedocs-hosted.com/en/latest/getting_started.html
 
+-------------
+Development
+-------------
+
+To make changes to the SDK you'll want to install it from source with the
+documentation and test dependencies:
+
+.. code-block:: bash
+
+    git clone https://github.com/pi-top/pi-top-Python-SDK.git
+    cd pi-top-Python-SDK
+    pip3 install -e ".[doc,test]"
+
+
+Changes you make to the source will be reflected in your Python environment.
+
+You may want to repeat this process for the pi-top-Python-Common-Library_ if
+it's not installed already or you need to make changes there too.
+
+.. _pi-top-Python-Common-Library: https://github.com/pi-top/pi-top-Python-Common-Library
+
+Once the SDK is installed you can run the automated test suite with:
+
+.. code-block:: bash
+
+    python3 -m pytest
+
+And you can build the docs locally by running:
+
+.. code-block:: bash
+
+    PYTHONPATH=. sphinx-build -W -v -bhtml docs/ build/html
```

Comment:
we should mention that `gpiozero` is a requirement to build the docs! Check this: https://github.com/pi-top/pi-top-Python-SDK/pull/126/commits/a260e817290e2d170d4884fd8a9a015f771cbbef ... If it's not on the system, the class docs won't be generated!

Reply:
Ah, interesting. But `gpiozero` would be installed by the above `pip3 install -e ".[doc,test]"` so I don't think we need to mention it explicitly.

Reply:
great!